src/anomalib/callbacks/post_processor.py

```diff
@@ -27,98 +138,15 @@ def on_validation_batch_end(
         self,
```

Comment:
Do we need this callback as `LightningModule` inherits from `ModelHooks` which have `on_validation_epoch_end` and `on_validation_batch_end`?

Reply:
This module is deprecated with the new design. I left it here for legacy purposes until we decide how to handle backward compatibility, but you can ignore it for now.